src/components/pages/dynamic-page.svelte

```diff
@@ -65,7 +65,6 @@
     story={page.story}
     blogIndex={page.blogIndex}
     projectsIndex={page.projectsIndex}
-    homePosts={page.homePosts}
```

Comment:
Blog posts are gone from the homepage

<img width="1100" alt="image" src="https://github.com/significa/significa.co/assets/98809371/2cc3326f-f589-4c7a-884f-fdbb198b7b05">

Reply:
I also get this in the console 
<img width="360" alt="image" src="https://github.com/significa/significa.co/assets/98809371/801dfab6-6275-4d35-a317-ec1c57f1332d">